neqo-transport/tests/server.rs

```diff
@@ -310,7 +309,17 @@ fn zero_rtt() {
     // The server will have received two STREAM frames now if it processed both packets.
     let active = server.active_connections();
     assert_eq!(active.len(), 1);
-    assert_eq!(active[0].borrow().stats().frame_rx.stream, 2);
+    assert_eq!(
+        active
+            .iter()
+            .next()
+            .unwrap()
+            .borrow()
+            .stats()
+            .frame_rx
+            .stream,
+        2
+    );
```

Comment:
`active_connections()` now returns a `HashSet` instead of a `Vec`. Thus `active.iter().first().unwrap()` instead of `active[0]`.